tests/sentry/replays/test_organization_replay_index.py

```diff
@@ -1869,3 +1872,52 @@ def test_exp_query_branches_error_ids_conditions(self):
                 response_data = response.json()
                 assert len(response_data["data"]) == 1, query
                 assert len(response_data["data"][0]["new_error_ids"]) == 1, query
+
```

Comment:
Needs coverage asserting sorts/filters work against these new fields.  There are two big test cases where I test all the filters and sorts.  You're welcome to append there or create new tests.  Whichever provides the best coverage.